src/main/java/racingcar/domain/RacingGame.java

```diff
@@ -1,43 +1,67 @@
 package racingcar.domain;
 
-import racingcar.strategy.MovingStrategy;
+import racingcar.common.Constant;
 import racingcar.domain.car.Car;
-import racingcar.domain.car.Name;
 import racingcar.domain.car.Cars;
+import racingcar.domain.car.Name;
+import racingcar.domain.strategy.MovingStrategy;
+import racingcar.domain.strategy.RandomMovingStrategy;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class RacingGame {
 
-    public Cars createCars(List<String> carNames) {
-        List<Car> cars = new ArrayList<>();
-        for (String carName : carNames) {
-            Name name = new Name(carName);
-            cars.add(new Car(name));
-        }
+    private final String racingCarNames;
+    private final int racingCount;
+    private static final String SPLIT_LETTER = ",";
```

Comment:
```suggestion
    private static final String SPLIT_LETTER = ",";

    private final String racingCarNames;
    private final int racingCount;
    
```
클래스 내부 변수의 선언순서는 상수먼저 선언합니다 !